andela/elven-ah
# #160344613 Fix console  errors and others

#### What does this PR do?
Fixes unhandled rejection errors in the console after a successful request and other fixes

#### Description of Task to be completed?
- fix unhandled promise errors in the console
- exclude untested files from coverage
- modify verification, password reset and auth callback links to reflect versioning
- add tests and remove extraneous tests
- modify responses to ensure consistent response objects
- remove test assertions for changeable messages
- make emails point to the originating host instead of the API

#### How should this be manually tested?
- Clone the repo: Run `git clone https://github.com/andela/elven-ah.git` in the folder where you want the repo to be saved.
- Navigate into the newly created folder and install the dependencies: `cd elven-ah && npm install`
- Copy .env.example to .env: `cp .env.example .env`
- Provision a PostgreSQL database and add the credentials to the .env file. 
- run test `npm test` or run the server and make requests.
- there should be no console errors

#### What are the relevant pivotal tracker stories? 
[#160344613](https://www.pivotaltracker.com/story/show/160344613)
